From 02ec52eefb237d7c2ebc65786328d533e5c4785a Mon Sep 17 00:00:00 2001
From: Angie Wang <angie.Wang@windriver.com>
Date: Fri, 8 Dec 2017 11:04:24 -0500
Subject: [PATCH 33/91] US106592:Pike Rebase.Update the keystonemiddle version
 in requirements.txt

keystonemiddle 4.19.0 breaks ceilometer tox since it's missing a dependency on python-memcached.
---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index b714a91..d8c848c 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -10,7 +10,7 @@ debtcollector>=1.2.0 # Apache-2.0
 jsonpath-rw-ext>=0.1.9 # Apache-2.0
 jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
 kafka-python>=1.3.2 # Apache-2.0
-keystonemiddleware!=4.1.0,>=4.0.0 # Apache-2.0
+keystonemiddleware!=4.19.0,>=4.12.0 # Apache-2.0
 lxml>=2.3 # BSD
 monotonic
 msgpack-python>=0.4.0 # Apache-2.0
-- 
2.7.4

